usr/src/cmd/cmd-inet/usr.sbin/ipsecutils/ipsecconf.c

```diff
@@ -3929,8 +3937,16 @@ parse_pattern_or_prop(FILE *fp, char *argvec[], char **leftover)
 				 * leftover buffer if any.
 				 */
 				if (*buf != '\0') {
-					(void) strlcpy(lo_buf, buf,
+					/*
```

Comment:
Suggestion:  Rip this out into a function:

```
/*
 * NB: src may point into part or all of dst. Let's play
 * it safe and use this as a wrapper to memmove().
 * XXX KEBE SAYS could return (len) just like strlcpy().
 */
void
strlcpy_move(char *dst, const char *src, size_t dstlen)
{
        size_t len = MIN(strlen(src), dstlen);
        
        memmove(dst, src, len);
}

Reply:
Ripped out more thoroughly, in a way that guarantees termination etc.